src/display.jl

```diff
@@ -430,16 +432,15 @@ Adds a video frame to the VideoStream `io`.
 """
 function recordframe!(io::VideoStream)
     #codec = `-codec:v libvpx -quality good -cpu-used 0 -b:v 500k -qmin 10 -qmax 42 -maxrate 500k -bufsize 1000k -threads 8`
-    frame = colorbuffer(io.screen)
-    _xdim, _ydim = size(frame)
-    xdim = _xdim % 2 == 0 ? _xdim : _xdim + 1
-    ydim = _ydim % 2 == 0 ? _ydim : _ydim + 1
-    frame_out = fill(RGB{N0f8}(1, 1, 1), ydim, xdim)
-    for x in 1:_xdim, y in 1:_ydim
-        c = frame[(_xdim + 1) - x, y]
-        frame_out[y, x] = RGB{N0f8}(c)
+    frame = colorbuffer(io.screen, ffmpeg_format = true)
+    _ydim, _xdim = size(frame)
+    if isodd(_xdim) || isodd(_ydim)
```

Comment:
Is it actually possible for this to be true? I tried setting the scene window size to an odd number, but the sizes here always returned even.
If something's coercing the dims to even, then this and paddedview dep can be removed

Reply:
Good question :D

Reply:
I wondered if it might be a retina display thing? I only tried it on MacOS

Reply:
Oh yeah, this is very likely platform dependent :D

Reply:
Cool. It would be interesting to see how much slower recording an odd-sized window would be

Reply:
Let me test today on win & linux... Would be nice to not have additional complexity for something that can't happen :D